Rocket.Chat/BlockedUsersListViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  BlockedUsersListViewController.swift
```

Comment:
Can you move this file to the correct folder? 

It's in Rocket.Chat/
Should be in Rocket.Chat/Controllers/Preferences